st2actions/st2actions/runners/pythonrunner.py

```diff
@@ -84,17 +86,10 @@ def _set_up_logger(self):
         Set up a logger which logs all the messages with level DEBUG
         and above to stderr.
         """
+        logging.config.fileConfig(action_config.get_logging_config_path())
         logger_name = 'actions.python.%s' % (self.__class__.__name__)
         logger = logging.getLogger(logger_name)
 
-        console = stdlib_logging.StreamHandler()
-        console.setLevel(stdlib_logging.DEBUG)
-
-        formatter = stdlib_logging.Formatter('%(name)-12s: %(levelname)-8s %(message)s')
-        console.setFormatter(formatter)
-        logger.addHandler(console)
-        logger.setLevel(stdlib_logging.DEBUG)
-
```

Comment:
The previous log levels were DEBUG for console logs.  This PR changes the log level to INFO for both console and pack logs.  This is appropriate in my opinion, just wanted to be sure we are aware.  
Do we want pack file logging enabled in Connect deployments?